www/install/steps/process/createDbUser.php

```diff
@@ -82,48 +82,132 @@
 
 // creating the user - mandatory for MySQL DB
 $alterQuery = "ALTER USER :dbUser@:host IDENTIFIED WITH mysql_native_password BY :dbPass";
-$query = "GRANT ALL PRIVILEGES ON `%s`.* TO " . $parameters['db_user'] . "@" . $host . " WITH GRANT OPTION";
+
+// Set defined privileges for the user.
+$mandatoryPrivileges = [
+    'SELECT',
+    'UPDATE',
+    'DELETE',
+    'INSERT',
+    'CREATE',
+    'DROP',
+    'INDEX',
+    'ALTER',
+    'LOCK TABLES',
+    'CREATE TEMPORARY TABLES',
+    'EVENT',
+];
+$privilegesQuery = implode(', ', $mandatoryPrivileges);
+$query = "GRANT " . $privilegesQuery . " ON `%s`.* TO " . $parameters['db_user'] . "@" . $host;
```

Comment:
since `%s` and `$parameters['db_user']` are comming from wizard inputs, could you use a prepare statement with bind values ?

Reply:
escaped in getConfiguration